# input topics
image_topic: "/camera/infra1/image_rect_raw"
image_compressed: true
camera_frame_name: "camera_infra1_optical_frame"

camera_info_topic: "/camera/infra1/camera_info"

lidar_topic: "/os_cloud_node/points"

# transformation form lidar to camera frame
# C_r_CL: [0.042, 0.077, -0.014] # tx ty tz
# C_q_CL: [-0.653, 0.271, -0.271, 0.653] # qx qy qz qw(real)

C_r_CL: [0.025, -0.014, -0.084] # tx ty tz
C_q_CL: [0.653, -0.271, 0.271, 0.653] # qx qy qz qw(real)
